documentation/docs/examples/java/introduction.mdx

```diff
@@ -0,0 +1,136 @@
+* [Deleting users](delete-users)
+* [Adding Trusted Authorities](trusted-authorities)
+
+
+### ChannelClient
+
+You can use the `ChannelClient` to access Audit Trail features. These include, but are not limited to:
+
+* [Creating a channel](create-channel)
+* [Subscribing or approving access to a channel](authorize-to-channel#subscribe-to-a-channel)
+* [Reading and Writing on a channel](create-channel#write-on-a-channel)
+
+
+You can import and configure these clients using a `env.properties` object which will populate the `BaseClient` class:
+
+ ```
+  api-key=XXXXXXX
+  api-version=vX.X
+  api-url=XXXXXXX
+  identity-file=adminIdentity.json
+```
+
+## Authorization
+
+In order to work with Integration Services API, you will need to be authenticated with a decentralized identity.
+
+The Integration Services use a JWT token based authorization mechanism which behaves in the following manner:
+
+1. The client gets a nonce from the API and returns it signed by the identity's private key.
+2. The API returns a JWT token to authorize any subsequent requests.
+
+You can get an identity using the following script (no auth required):
+
+```java
+  IdentityClient client = new IdentityClient();
+  JSONObject jsonClaim = new JSONObject().put("type", "Person").put("name", "randomName");
+
+  Claim claim = new Claim(jsonClaim);
+
+  final String randomUsername = Utils.getRandomUsernameOfLength(5);
+  System.out.println("username: " + randomUsername);
+  JSONObject newUserIdentity = client.create(randomUsername, claim);
+  System.out.println("created new user " + newUserIdentity);
+
+```
+
+The generated identity is stored on the IOTA Tangle and follows [did-core specifications](https://www.w3.org/TR/did-core/).
+It is a JSON file like the following:
+
+```json
+{
+  "doc": {
+    "id": "did:iota:3q4mW831dfbbSP2j5Lf4FPTKXFjEv2ykyPjuVrd1FHss",
+    "authentication": [
+      {
+        "id": "did:iota:3q4mW831dfbbSP2j5Lf4FPTKXFjEv2ykyPjuVrd1FHss#key",
+        "controller": "did:iota:3q4mW831dfbbSP2j5Lf4FPTKXFjEv2ykyPjuVrd1FHss",
+        "type": "Ed25519VerificationKey2018",
+        "publicKeyBase58": "5LT5yjaykKeTqYuqmCELE7xUyN9z4WEN5CKxdWmrU5g5"
+      }
+    ],
+    "created": "2021-12-20T10:43:00Z",
+    "updated": "2021-12-20T10:43:00Z",
+    "proof": {
+      "type": "JcsEd25519Signature2020",
+      "verificationMethod": "#key",
+      "signatureValue": "ktusPiZd5whHTHsJJwSiSbfhmAGypdkTgLFQt4USvVg91WJLYNhEzshcBCvr9Cr42heyB249TvZHVrdGVAYds6s"
+    }
+  },
+  "key": {
+    "type": "ed25519",
+    "public": "5LT5yjaykKeTqYuqmCELE7xUyN9z4WEN5CKxdWmrU5g5",
+    "secret": "9Arv8HTAu3JjLvBhst6deMaULvBc3e7w2WiseEL4BESC",
+    "encoding": "base58"
+  }
+}
+```
+
+If you have a JSON Identity, you can authorize your client as follows:
+
+```java
+  client.authenticate(newUserIdentity.getJSONObject("doc").getString("id"),
+			  newUserIdentity.getJSONObject("key").getString("public"),
+				newUserIdentity.getJSONObject("key").getString("secret"));
```

Comment:
Would it make sense to introduce local variables to make the example more readable?